src/node_jsvmapi.cc

```diff
@@ -1002,28 +1002,29 @@ napi_status napi_create_array_with_length(napi_env e, int length, napi_value* re
   return GET_RETURN_STATUS();
 }
 
-napi_status napi_create_string(napi_env e, const char* s, napi_value* result) {
+napi_status napi_create_string_utf8(napi_env e, const char* s, size_t length,
```

Comment:
Oh, I need to either change `size_t length` back to `int` or restore the without-length variant for creating a string value. My intention was that length here could be -1 when the length is not known by the caller, except `size_t` is unsigned.